### PR TITLE
Validator manegement stories and spacing fixes

### PR DESCRIPTION
**Pull request by HristoNedelkov**

### CODE REVIEW

**Review by RadoslavDimchev — CHANGES_REQUESTED** [review]
Why are you imported old things for manage validator?

File: src/pages/ManageValidator/LeftManage/ValidatorBlueSection/ValidatorBalance.tsx [hunk 1]
```diff
@@ -29,8 +32,8 @@ const ValidatorBalance = () => {
           $0 USD
         </Text>
       </YStack>
-      <Stack marginRight="5rem">
-        <Icon src="/icons/suit-guy.png" width={200} height={70} />
+      <Stack marginRight="3rem">
+        <Icon src="/icons/suit-guy.png" width={200} height={70}></Icon>
```
Comment (EmilIvanichkovv) [inline src/pages/ManageValidator/LeftManage/ValidatorBlueSection/ValidatorBalance.tsx:36]: Ditto. This was changed [here](https://github.com/nimbus-gui/nimbus-gui/commit/6a92e4e0b6709162d05969d85faaefad7a293af1#diff-31ee1c410505317957ce3bb2ecd8716bf266b54b65a4149f33f32779a6fe7c0e)

File: src/pages/ManageValidator/LeftManage/ValidatorBlueSection/ValidatorBlueSection.tsx [hunk 2]
```diff
@@ -1,16 +1,12 @@
...
 import ValidatorBalance from './ValidatorBalance'
 
 const ValidatorBlueSection = () => {
   return (
-    <YStack
-      backgroundColor="#F4F6FE"
-      space={'$3'}
-      style={{ padding: '30px 15px 15px 30px' }}
-    >
+    <YStack backgroundColor="#F4F6FE" space={'$2'}>
```
Comment (EmilIvanichkovv) [inline src/pages/ManageValidator/LeftManage/ValidatorBlueSection/ValidatorBlueSection.tsx:9]: This is a change introduced by @RadoslavDimchev [here](https://github.com/nimbus-gui/nimbus-gui/commit/e76b4f00a2050b30c728bc8878207adc1af86819) why do you revert it?

**Review by EmilIvanichkovv — CHANGES_REQUESTED** [review]
There seem to have a lot of reverts of @RadoslavDimchev work from the last days. Please sync with him to clear what out

**Pull request closed without merging**